core/key.go

```diff
@@ -41,14 +41,17 @@ const (
 )
 
 const (
-	COLLECTION                = "/collection/names"
-	COLLECTION_SCHEMA         = "/collection/schema"
-	COLLECTION_SCHEMA_VERSION = "/collection/version"
-	COLLECTION_INDEX          = "/collection/index"
-	SEQ                       = "/seq"
-	PRIMARY_KEY               = "/pk"
-	REPLICATOR                = "/replicator/id"
-	P2P_COLLECTION            = "/p2p/collection"
+	COLLECTION                        = "/collection/names"
+	COLLECTION_SCHEMA                 = "/collection/schema"
+	COLLECTION_SCHEMA_VERSION         = "/collection/version/v"
+	COLLECTION_SCHEMA_VERSION_HISTORY = "/collection/version/h"
```

Comment:
We don't have a history of usage though so I don't think it would be confusing in that case.

Reply:
We didnt have a history of version until this PR either.  And just because something doesnt already exist within the codebase doesnt mean it does not exist in the reader's head.